internal/k8s/controller_test.go

```diff
@@ -1382,6 +1382,56 @@ func TestFindVirtualServerRoutesForService(t *testing.T) {
 	}
 }
 
+func TestCheckForVirtualServerRoute(t *testing.T) {
```

Comment:
the name of the unit test needs to reflect the name of the function under the test.